script.js

```diff
@@ -0,0 +1,81 @@
+const container = document.querySelector('.mobile-header-nav-links');
+const icon = document.querySelector('.menu');
+const times = document.querySelector('.close');
+
+function open() {
+  container.style.display = 'flex';
+}
+
+function close() {
+  container.style.display = 'none';
+}
+
+icon.addEventListener('click', open);
+times.addEventListener('click', close);
+
+// Dynamic Speakers
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.